test/ViewExtensionLibraryTests/resources/libraryItems.json

```diff
@@ -369,9 +369,9 @@
       "keywords": "Dynamo.Nodes.dyn3DPreview, Dynamo.Nodes.3DPreview, Dynamo.Nodes.Watch3D, DynamoWatch3D.Watch3D, Watch 3D"
     },
     {
-      "fullyQualifiedName": "Rank",
-      "iconUrl": "http://localhost/icons/Rank.Small?path=BuiltIn",
-      "contextData": "Rank@var[]..[]",
+      "fullyQualifiedName": "List.Rank",
+      "iconUrl": "http://localhost/icons/List.Rank.Small?path=BuiltIn.ds",
+      "contextData": "List.Rank@var[]..[]",
       "parameters": null,
```

Comment:
@aparajit-pratap Check out the iconUrl for LoopWhile [here](https://github.com/aparajit-pratap/Dynamo/blob/c8108df84e65397be0fab49a6cb8b3c669a42aa9/test/ViewExtensionLibraryTests/resources/libraryItems.json#L381).  The `VerifyIconsForLibraryItems` test passes while the iconUrl still links to `BuiltIn` but fails when changed to `BuiltInNodes`.  This makes me think that the correct icon is being found in `BuiltIn`

Reply:
When the test loads an `svg` icon it fails because it is loading the default dynamo icon.  If the test passes it find the correct `png` associated with the specified location above